rest/taskana-rest-spring-test/src/test/java/pro/taskana/jobs/AsyncUpdateJobIntTest.java

```diff
@@ -58,86 +51,75 @@
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(AsyncUpdateJobIntTest.class);
-    String server = "http://127.0.0.1:";
-    RestTemplate template;
-    HttpEntity<String> request;
-    HttpHeaders headers = new HttpHeaders();
-    @LocalServerPort
-    int port;
+    @Autowired JobScheduler jobScheduler;
+
+    @Autowired Environment env;
+
+    @LocalServerPort int port;
+
+    private String server;
+
+    private RestTemplate template;
 
     @Before
     public void before() {
         template = getRestTemplate();
-        headers.add("Authorization", "Basic dGVhbWxlYWRfMTp0ZWFtbGVhZF8x");
-        request = new HttpEntity<String>(headers);
+        server = "http://127.0.0.1:" + port;
     }
 
     @Test
     public void testUpdateClassificationPrioServiceLevel()
-        throws IOException, InterruptedException, InvalidArgumentException {
+        throws IOException, InvalidArgumentException {
 
         // 1st step: get old classification :
         Instant before = Instant.now();
         ObjectMapper mapper = new ObjectMapper();
-        String updatedClassification;
 
         ResponseEntity<ClassificationResource> response = template.exchange(
-            "http://127.0.0.1:" + port + "/api/v1/classifications/CLI:100000000000000000000000000000000003",
+            server + "/api/v1/classifications/{classificationId}",
             HttpMethod.GET,
-            request,
-            ParameterizedTypeReference.forType(ClassificationResource.class));
+            new HttpEntity<String>(getHeaders()),
+            ParameterizedTypeReference.forType(ClassificationResource.class),
+            CLASSIFICATION_ID);
 
-        assertNotNull(response.getBody().getLink(Link.REL_SELF));
+        assertNotNull(response.getBody());
         ClassificationResource classification = response.getBody();
+        assertNotNull(classification.getLink(Link.REL_SELF));
 
         // 2nd step: modify classification and trigger update
         classification.removeLinks();
         classification.setServiceLevel("P5D");
         classification.setPriority(1000);
 
-        updatedClassification = mapper.writeValueAsString(classification);
-
-        URL url = new URL(server + port + "/api/v1/classifications/CLI:100000000000000000000000000000000003");
-        HttpURLConnection con = (HttpURLConnection) url.openConnection();
-        con.setRequestMethod("PUT");
-        con.setRequestProperty("Authorization", "Basic dGVhbWxlYWRfMTp0ZWFtbGVhZF8x");
-        con.setDoOutput(true);
-        con.setRequestProperty("Content-Type", "application/json");
-        BufferedWriter out = new BufferedWriter(new OutputStreamWriter(con.getOutputStream()));
-        out.write(updatedClassification);
-        out.flush();
-        out.close();
-        assertEquals(200, con.getResponseCode());
-        con.disconnect();
+        template.put(
+            server + "/api/v1/classifications/{classificationId}",
```

Comment:
extract /api/v1/classifications to constant?

Reply:
next step ...  want to improve urls on another ticket